.gitignore

```diff
@@ -13,3 +13,7 @@ __pycache__/
 .vscode/
 .devcontainer/
 .coverage
+
+#Virtual environment and IDE files
+venv/
```

Comment:
Why we need this if we suggest to use `~/keras-cv-venv`?

Reply:
Often time IDEs don't create `venv` in the root directory, especially if you're working on Windows. And since the default name goes with `venv` I added it (actually I do have `venv` directory in my local setup too).